pkg/bootstrap/manifests/klusterlet/bootstrap_secret.yaml

```diff
@@ -14,6 +14,19 @@ type: Opaque
 data:
   kubeconfig: "{{ $bootstrapkubeconfigsecret.KubeConfig }}"
 {{end}}
+---
+apiVersion: v1
+kind: Secret
```

Comment:
TBH I think we should resolve this in code. I would expect you put it in the BootstrapKubeConfigSecrets list and also remove any duplications in the list

Reply:
Do we mean modify the `KlusterletConfig.BootstrapKubeConfigSecrets` directly here? KlusterConfig needs to bind with managed cluster via annotation, does this mean we need to reconcile every KlusterletConfig and add default bootstrapkubeconfig  of current hub?

Reply:
yes when klusterletConfig has this part set.

Reply:
We can set it to the klusterletConfig, because the `KubeConfigSecret` in `KlusterletConfig` only has the secret name, and the secret is created in the same namespace with import-controller. 

Not like default bootstrapKubeConfig which data part is generated or decoded from an import-secret. 

So I still put it into the `render.go`, but in a middle stage list -- `bootstrapKubeConfigSecrets`, in the future if we want to add deduplication, we can do it based on this stage.